src/server/controllers/login.js

```diff
@@ -19,8 +19,7 @@ module.exports = (req, res) => {
         res.send({ err: err.message });
       }
       if (isAuthenticated) {
-        // need to set cookie
-        res.redirect('/');
+        res.cookie('name', login, { maxAge: 3600000 }).redirect('/');
```

Comment:
Is this one hour?

I would write it as 1000 * 60 * 60, otherwise it is hard to know what it means